pkg/server/server.go

```diff
@@ -106,6 +106,7 @@ type jsGlobals struct {
 	QuickStarts                string   `json:"quickStarts"`
 	ProjectAccessClusterRoles  string   `json:"projectAccessClusterRoles"`
 	Clusters                   []string `json:"clusters"`
+	ControlPlaneToplogy        string   `json:"controlPlaneToplogy"`
```

Comment:
@jhadvig - The flag value is set in the jsGlobals already.  Is this what you meant?  I do see there is a typo in the name though throughout my code which I will fix shortly.

Reply:
Right but I dont see where we are registering the flag in the `main.go`